### ISSUE TITLE
Electrolyser Management based on PRICE, uses excessElectricPower_kW, is this correct?? CHECK AND FIX

### ISSUE THREAD

**Issue by AteZenmo** [Zenmo/zero_engine#51]

**Comment by GillisHommen** [comment]
@AteZenmo wat is hier de status van? Als'ie niet meer relevant is graag sluiten! :)

**Comment by AteZenmo** [comment]
denk nog wel relevant, zal er binnenkort naar kijken

**Comment by GillisHommen** [comment]
Nog updates over dit issue?

**Comment by AteZenmo** [comment]
Ja, even naar gekeken: komt er op neer dat die functie helemaal niet gebruikt wordt nog. (Prijs aansturing is nog redelijk basic, iemand heeft daar gewoon de excess gepakt om de prijs op te halen bij de connection owner, maar vervolgens wordt die prijs van de connection owner helemaal nog niet gebruikt. Maar wordt gewoon nog steeds de EPEX prijs gepakt om te sturen. Mocht prijs aansturing ooit verbeterd moeten worden en slimmer gemaakt moeten worden voor een project stel ik voor om het dan eventueel te verbeteren. Voor nu: de prijs aansturing voor de electrolyser in een GCConversion werkt gewoon nog steeds naar behoren. Deze issue kan dus sluiten.
